packages/snowpack/src/types/snowpack.ts

```diff
@@ -0,0 +1,198 @@
+import type HttpProxy from 'http-proxy';
+import {Plugin as RollupPlugin} from 'rollup';
+
+export type DeepPartial<T> = {
```

Comment:
The new types file. FWIW I usually prefer explicit imports/exports like this, rather than global declarations. I’ve found global declarations to be a PITA when other people `npm install` your package for its types, and TypeScript can’t resolve things.

Reply:
+1000000000000